### DISCUSSION TITLE
apollo源码如何可以打成一个quick start包和如何快速启动所有服务。

### DISCUSSION THREAD

**Discussion by small-fresh-meet** [apolloconfig/apollo-java#38] [Q&A]
我自己按照文档quick start方式打包之后，在assembly下面生成一个jar包。首先，这不是一个quick start包，第一个问题就是源码如何可以打成一个quick start包。第二个问题是，默认assembly下面的jar包，应该如何直接启动就可以把所有服务都启动起来，里面的数据库参数要怎么配置，是直接修改properties之后再打包吗？还是有其他方式。

**Comment by nobodyiam** [comment]
* [1.3.1 手动打包Quick Start安装包](https://www.apolloconfig.com/#/zh/deployment/quick-start?id=_131-%e6%89%8b%e5%8a%a8%e6%89%93%e5%8c%85quick-start%e5%ae%89%e8%a3%85%e5%8c%85)
* [2.2 配置数据库连接信息](https://www.apolloconfig.com/#/zh/deployment/quick-start?id=_22-%e9%85%8d%e7%bd%ae%e6%95%b0%e6%8d%ae%e5%ba%93%e8%bf%9e%e6%8e%a5%e4%bf%a1%e6%81%af)

**Reply by small-fresh-meet** [reply]
之前按这个方法试过 ，但是按这个命令打成的quick start包 ，无法直接用demo.sh来启动，会报错，找不到config-service。
